__tests__/main.test.ts

```diff
@@ -16,13 +15,14 @@ jest.mock('../src/AzureSqlResourceManager');
 jest.mock('../src/Setup');
 
 describe('main.ts tests', () => {
-    afterEach(() => {
+    beforeEach(() => {
         jest.restoreAllMocks();
-    })
+        jest.clearAllMocks();
+    });
 
     it('gets inputs and executes build and publish action', async () => {
         const resolveFilePathSpy = jest.spyOn(AzureSqlActionHelper, 'resolveFilePath').mockReturnValue('./TestProject.sqlproj');
-        const getInputSpy = jest.spyOn(core, 'getInput').mockImplementation((name, options) => {
+        let getInputSpy = jest.spyOn(core, 'getInput').mockImplementation((name, options) => {
```

Comment:
why did this get changed from const to let?

Reply:
Changed back